Lib/urllib/request.py

```diff
@@ -2087,20 +2087,26 @@ def proxy_bypass_registry(host):
             return False
         if not proxyEnable or not proxyOverride:
             return False
-        return _proxy_bypass_winreg_override(host, proxyOverride)
 
-    def proxy_bypass(host):
+        return proxyOverride
+
+    def proxy_bypass_registry(host, proxyOverride=None):
```

Comment:
I do see that some variables were named with camelCase above, but I'd prefer to not expose that in function args. Let's change this to `proxy_override`.